airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -1978,6 +1978,10 @@ definitions:
           - "$ref": "#/definitions/SelectiveAuthenticator"
           - "$ref": "#/definitions/CustomAuthenticator"
           - "$ref": "#/definitions/LegacySessionTokenAuthenticator"
+      fetch_properties_from_endpoint:
```

Comment:
I know we were flagging things as experimental in the CDK until they were added to the Connector Builder and that with Lake's project, it is not necessarily possible anymore as they would be automatically added. Is there still a concern here of flagging this as a feature that is new?

Reply:
I think since its optional and non-breaking I'm fine leaving it a bit undocumented as new. I'm not super concerned for that reason. I don't expect many customer made connectors to need to enable this

Maybe there's a broader questions about how we socialize new features, but maybe we kick that can down the road.